pkg/validation/validation.go

```diff
@@ -81,6 +81,10 @@ func NodeGroups(nodeGroups []nropv1.NodeGroup, platf platform.Platform) error {
 		return err
 	}
 
+	if err := nodeGroupsAnnotations(nodeGroups); err != nil {
```

Comment:
can be simplified to
`return nodeGroupsAnnotations(nodeGroups)`

Reply:
rearranged the function to simplify further